lib/miq_tools_services/github/message_builder.rb

```diff
@@ -0,0 +1,56 @@
+require 'stringio'
+
+module MiqToolsServices
+  class Github
+    class MessageBuilder
```

Comment:
I agree, but I'm having trouble naming.  In my mind when I wrote it on the bot, a message was the singular, complete thing you wanted to build, and it is made up of one or more comments.

On that note, the `.messages` method is a bad name, it really should be `.comments`. I can change that.